test/testapp.go

```diff
@@ -50,29 +52,31 @@ func (a *testApp) UniqueID() string {
 	return "testApp" // TODO should this be randomised?
 }
 
-func (a *testApp) applyThemeTo(content fyne.CanvasObject, canv fyne.Canvas) {
+func (a *testApp) applyThemeTo(content fyne.CanvasObject) {
 	if content == nil {
 		return
 	}
 	content.Refresh()
 
-	if wid, ok := content.(fyne.Widget); ok {
-		for _, o := range wid.CreateRenderer().Objects() {
-			a.applyThemeTo(o, canv)
+	switch x := content.(type) {
+	case fyne.Widget:
+		for _, o := range cache.Renderer(x).Objects() {
```

Comment:
Wouldn't `test.WidgetRenderer()` be better, not adding dependency on cache package.

Reply:
Well, it would not really reduce the dependency of the package but okay.

Reply:
I meant that this has a new dependency on cache, that line of code would go away - hiding the internal details.